.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,24 @@
+name: Check the source code
+on:
+  pull_request:
+jobs:
+  test:
+    name: Check the source code
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install Yarn
+        run: npm install -g yarn
```

Comment:
when you use, actions/checkout, i think already yarn install process will be done.
maybe can you play around first by setting `on: manual` ?
I will ask to add you this org so I think you will be able to do so. just wait a moment

Reply:
I was verifying using [act](https://github.com/nektos/act) locally and was unable to use yarn in actions/setup-node, so I am writing this based on the following comment.

https://github.com/actions/setup-node/issues/182#issuecomment-966885975

I should have written a comment because it's hard to understand.